bundles/net.tourbook/src/net/tourbook/data/CustomTrackDefinition.java

```diff
@@ -8,31 +8,33 @@
     */
    private static final long serialVersionUID = -4880414084528375769L;
    //Custom Track Definition
-   private String Name;
-   private String Id;
-   private String Unit;
+
+   private String            _Name;
+   private String            _Id;
+   private String            _Unit;
```

Comment:
Eclipse discourages the use of an uppercase character for a field name, in this case these fields have a _ (underscrore) prefix therefore this is warning would not be displayed.

Only constant fields (static final fields) should start with an Uppercase or the whole name is UPPERCASE but "normal" fields should start with a lowercase (when ignoring the _ )

![mt-uppercase-fields](https://user-images.githubusercontent.com/1283445/103664974-270e1600-4f73-11eb-975b-b509df85a291.png)

Reply:
well, I didn't know what to do precisely with github.
So I created another pull request which should contain everything.

how can I modify a file without creating a Pull request actually ??

Reply:
As this Pull request is still open, pushing your commit will update the PR, did you try ?

Reply:
that's what I did, I first pushed to main branch, but it didn't seem to update the PR.

Reply:
or it did and I didn't realize :-) my update is "custom track #6c"
I can see it below the comments here, but not sure that it is part of the current PR ?

Reply:
Ok, i see the issue.
You did push your commit to your main but you created a different PR with Wolfgang's main branch (PR should not be done to Wolfgang's main branch)
Essentially, you don't need to create a new PR. So cancel this [one](https://github.com/wolfgang-ch/mytourbook/pull/281) as the current one has your latest [commit ](https://github.com/wolfgang-ch/mytourbook/pull/280/commits/781b4311423a9f01e009323a0f60833db5413af7)

I hope that helps :-)

Reply:
many thanks for the help, I canceled #281 .
and #281  was wrong anyway because it was on the wrong branch it should have been on "contributor-gmngueko :-) )